tests/schema.json

```diff
@@ -3,6 +3,10 @@
         "_meta": {
             "topic": "mz_datagen_users",
             "key": "id",
+            "proto": {
```

Comment:
Same comment. Let’s not change existing tests to require proto

Reply:
I am adding a `schema-proto.json` file to use instead